docs/pages/setup/admin/daemon.mdx

```diff
@@ -31,117 +35,174 @@ $ readlink /sbin/init
 
 </Notice>
 
-## Step 1/3. Install and configure Teleport
+## Install and configure Teleport
 
-Choose the appropriate instructions for your environment.
+If you have not already, select and complete the appropriate installation instructions for your environment.
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Teleport requires a configuration file to run. After installation, no
-configuration file exists. We will create a minimal configuration file to show
-you how to run Teleport as a daemon:
+Teleport requires a [configuration yaml file](https://goteleport.com/docs/setup/reference/config/) to be created and configured in order to run. After initial installation, no
+configuration file exists until it is manually created by a user. This guide will use a minimal teleport configuration based on defaults, however in production environments we
+recommend [reviewing configuration file options](https://goteleport.com/docs/setup/reference/config/) to better suit your own specific needs.
+
+To create the default teleport configuration file, enter the following command:
 
 ```code
 $ sudo teleport configure -o file
 Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
 ```
 
-## Step 2/3. Create a systemd unit file
+## Creating and Configuring the systemd Daemon
+
+Once you've initiially installed the `teleport` binary and created a Teleport configuration file, the Teleport daemon must be created and configured. While a daemon can be created and configured manually, the
```

Comment:
```suggestion
Once you've initially installed the `teleport` binary and created a Teleport configuration file,  let's create and configure Teleport daemon.
```

Reply:
nit: switched from passive to active and shortened the sentence according to [On writing well](https://www.goodreads.com/book/show/53343.On_Writing_Well)

Reply:
Searched for other opportunities to switch to the active voice, including your request and some others